### PR TITLE
Add Github Actions

### PR DESCRIPTION
**Pull request by GeertJohan**
Fixes go-flutter-desktop/go-flutter#400

### CODE REVIEW

**Comment by jld3103** [comment]
Github Actions can also run on Docker images (https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idcontainerimage)

File: stocks/go/go.mod [hunk 1]
```diff
@@ -1,6 +1,6 @@
-module github.com/go-flutter-desktop/examples/stocks/desktop
+module stocks/go
```
Comment (GeertJohan) [inline stocks/go/go.mod:1]: Just wondering why the prefix was removed?

File: xbox_controller/.flutter-plugins-dependencies [hunk 2]
```diff
@@ -0,0 +1 @@
...,"path":"/home/geertjohan/.pub-cache/hosted/pub.dartlang.org/audioplayers-0.15.1/","dependencies":["path_provider"]},{"name":"path_provider","path":"/home/geertjohan/.pub-cache/hosted/pub.dartlang.org/path_provider-1.6.5/","dependencies":[]}],"android":[{"name":"audioplayers","path":"/home/geertjohan/.pub-cache/hosted/pub.dartlang.org/audioplayers-0.15.1/","dependencies":["path_provider"]},{"name":"path_provider","path":"/home/geertjohan/.pub-cache/hosted/pub.dartlang.org/path_provider-1.6.5/","dependencies":[]}],"macos":[{"name":"audioplayers","path":"/home/geertjohan/.pub-cache/hosted/pub.dartlang.org/audioplayers-0.15.1/","dependencies":[]},{"name":"path_provider_macos","path":"/home/geertjohan/.pub-cache/hosted/pub.dartlang.org/path_provider_macos-0.0.4/","dependencies":[]}],"linux":[],"windows":[],"web":[{"name":"audioplayers","path":"/home/geertjohan/.pub-cache/hosted/pub.dartlang.org/audioplayers-0.15.1/","dependencies":[]}]},"dependencyGraph":[{"name":"audioplayers","dependencies":["path_provider"]},{"name":"path_provider","dependencies":["path_provider_macos"]},{"name":"path_provider_macos","dependencies":[]}],"date_created":"2020-04-19 16:16:22.027096","version":"1.17.0"}
```
Comment (GeertJohan) [inline xbox_controller/.flutter-plugins-dependencies:1]: I think we should ignore this file.

**Comment by GeertJohan** [comment]
I can't add a review in GitHub because I opened this PR. LGTM except for one file that shouldn't be checked in.

File: stocks/go/go.mod (hunk 1 above)
Comment (jld3103) [reply stocks/go/go.mod:1]: Oh I just re inited the stocks app, but forgot to provide the module. Will fix that

File: xbox_controller/.flutter-plugins-dependencies (hunk 2 above)
Comment (jld3103) [reply xbox_controller/.flutter-plugins-dependencies:1]: Yes

**Pull request merged**